src/main/java/org/grouplens/grapht/InjectionContainer.java

```diff
@@ -81,7 +82,8 @@ private InjectionContainer(CachePolicy dft) {
      * @see #makeInstantiator(DAGNode, SetMultimap)
      */
     public Instantiator makeInstantiator(DAGNode<Component, Dependency> node) {
-        return makeInstantiator(node, ImmutableSetMultimap.<DAGNode<Component, Dependency>, DAGEdge<Component, Dependency>>of());
+        Instantiator inst  =  makeInstantiator(node, ImmutableSetMultimap.<DAGNode<Component, Dependency>, DAGEdge<Component, Dependency>>of());
+        return inst;
```

Comment:
This change does not look necessary to me - it looks like all it is doing is introducing a local variable that is immediately returned. If we do want a variable here, let's just use one space on each side of `=`.